scripts/apps/monitoring/views/monitoring-view.html

```diff
@@ -109,35 +125,34 @@ <h3 class="subnav__page-title sd-flex-no-grow sd-empty">
                             </div>
                         </div>
                     </div>
-    
+
                     <div ng-hide="hideMonitoringToolbar2 === true" sd-media-query min-width="700" class="subnav">
             
                         <!-- filtering by file type not implemented for usage with custom data source -->
                         <div ng-if="(!elementState || elementState === 'comfort') && customDataSource == null" class="button-list button-list--padded">
-                            <a href="" ng-repeat="fileType in aggregate.fileTypes" ng-click="aggregate.setFileType(fileType)" class="toggle-button" ng-class="{'toggle-button--active': aggregate.hasFileType(fileType)}" title="{{fileType| translate}}">
-                                <span ng-if="fileType === 'all'" class="toggle-button__text toggle-button__text--all">{{:: 'all' | translate}}</span>
+                            <a href="" ng-repeat="fileType in aggregate.fileTypes" ng-click="aggregate.setFilterType('file', fileType)" class="toggle-button" ng-class="{'toggle-button--active': aggregate.hasFileType(fileType)}" title="{{ gettext(fileType) }}">
```

Comment:
don't pass dynamic values to `gettext`. I know it's been here like that, but if you are working on this part, it'd be good to fix it